src/components/pages/AdminDashboard/DevTools/DayComponent.js

```diff
@@ -17,6 +17,19 @@ const DayComponent = ({ day, devMode, setDevMode }) => {
     setDevMode(!devMode.isDevModeActive);
   };
 
+  const findNextDayOfWeek = selectedDay => {
+    let dayOfWeek = selectedDay;
+    let date = new Date();
+    let resultDate = new Date(date.getTime());
+    resultDate.setDate(date.getDate() + ((7 + dayOfWeek - date.getDay()) % 7));
+    console.log(resultDate);
```

Comment:
Should remove this console log before merging if we're certain the function is working